fetch.bs

```diff
@@ -2172,6 +2236,132 @@ exploits pertain to those <a for=request>types</a>. Also, considering "<code>ima
 compatible with deployed content.
 
 
+<h3 id=token-binding>Token Binding</h3>
+<p>In order to protect security tokens like <a>credentials</a> and OAuth tokens, user agents
+and servers can use Token Binding to cryptographically associate a given
+security token with the user agent, thereby making it more difficult for
+attackers to abuse the capabilities the security token grants.
+This method of associating credentials with a user agent is referred to as
+<dfn export id=concept-token-binding>token binding</dfn>.
+
+<p>The technique is described in
+detail in [[TOKBIND-NEGOTIATION]], [[TOKBIND-PROTOCOL]], and [[TOKBIND-HTTPS]].
+
+<p>A <dfn export id=token-binding-id>token binding ID</dfn> is the representation
+of a <a for=/>token binding</a>, as described in
+<a href="https://tools.ietf.org/html/draft-ietf-tokbind-protocol#section-3.2">section 3.2</a>
+of [[TOKBIND-PROTOCOL]].
+
+<p>At a very high level, a user agent negotiates the use of Token Binding when it
+sets up a TLS connection to a server.
+The user agent maintains a Token Binding key store, where it saves different
+<a for=/>token binding ID</a>s to be used with different servers.
+It maintains <a for=/>token binding ID</a>s at the granularity of
+effective top-level domain (public suffix) + 1 (eTLD+1).
+If the user agent and server agree to use Token Binding
+and a <a for=/>token binding ID</a> does not already exist for that server,
+the user agent generates a <a for=/>token binding ID</a> (essentially, a public-private key-pair)
+for use with the server,
+using TLS connection metadata saved at the end of the Token Binding Negotiation,
+and saves it in the Token Binding key store.
+Later, the user agent simply looks up the a <a for=/>token binding ID</a> whenever needed.
+When the user agent sends a <a for=/>request</a> to the server over the TLS connection,
+it proves possession of its <a for=/>token binding ID</a> to the server.
+The server associates credentials that it issues to that user agent with that
+<a for=/>token binding ID</a>,
+and verifies if credentials presented to it by a user agent match the
+<a for=/>token binding ID</a> used when the credentials were issued to that user agent.
+<p class="note no-backref">
+If the <a for=/>token binding ID</a> for a server is
+deleted by a user agent (e.g., as a result of some user action), the user agent will
+need to re-establish a TLS connection to negotiate Token Binding and create a new
+<a for=/>token binding ID</a>.
```

Comment:
Not sure this note is necessary here; seems like something to note in the token binding specs themselves.